### PR TITLE
Improve luacheck configuration

### PR DESCRIPTION
**Pull request by TomyLobo**

### CODE REVIEW

**Comment by TomyLobo** [comment]
The shell script still needs a review, btw

**Comment by TomyLobo** [comment]
The shell script still needs a review, btw
@Nebual @AbigailBuccaneer @Divran know anyone that could review this?

**Comment by AbigailBuccaneer** [comment]
Thanks for your work on this @TomyLobo ! I'll try and sneak in a review of the shell script tomorrow.

File: generate-luacheck.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                    :
+                ;;
+
+                *)
+                    echo >&2 "Warning: Unhandled line '$line'"
+            esac
+        done
+
+        printf '\n'
+    }        |
```
Comment (TomyLobo) [inline generate-luacheck.sh]: oops, lemme fix those extra spaces before you merge :)

Comment (TomyLobo) [inline generate-luacheck.sh]: I forgot, this was already merged. I fixed it on master.

**Pull request merged**